### PR TITLE
Adding Ghost and Drupal Platform Chart In Okteto Cloud Catalog

### PR DESCRIPTION
**Pull request by genievot**
Hello,
I am adding Ghost CMS Chart to Okteto cloud catalog.
The **icon** field in `Chart.yaml` the file is not working locally, But a valid URL is provided and you would like to make any modifications and make it more Okteto friendly, So if anything to comment please feel free to do so :)

### CODE REVIEW

File: ghost/config.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+ghostEmail: change-me@okteto.com
+ghostUsername: okteto
+ghostPassword: okteto12345
+allowEmptyPassword: false
```
Comment (pchico83) [inline ghost/config.yaml]: Move to `values.yaml`

File: ghost/config.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+ghostBlogTitle: User's Blog!
+
+ghostProtocol: https
```
Comment (pchico83) [inline ghost/config.yaml]: Move to `values.yaml`

File: ghost/config.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+resources:
+  limits:
+    memory: 2Gi
+    cpu: 1
+
+updateStrategy:
+  type: Recreate
+
+service:
```
Comment (pchico83) [inline ghost/config.yaml]: Move to `values.yaml`

File: ghost/templates/ingress.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
+{{- if .Values.ingress.enabled }}
```
Comment (pchico83) [inline ghost/templates/ingress.yaml]: You can remove this file, it is never used in Okteto Cloud

File: ghost/values.yaml [hunk 5]
```diff
@@ -0,0 +1,294 @@
...
+##
+image:
+  registry: docker.io
+  repository: bitnami/ghost
+  tag: 3.8.0-debian-10-r0
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
```
Comment (pchico83) [inline ghost/values.yaml]: you can remove this one, Okteto Cloud always overwrite this value to `PullAlways`

File: ghost/values.yaml [hunk 6]
```diff
@@ -0,0 +1,294 @@
...
+    ## Secrets must be manually created in the namespace.
+    ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+    ##
+    # pullSecrets:
+    #   - myRegistryKeySecretName
+
+## Ghost protocol, host, port and path to create application URLs
+## ref: https://github.com/bitnami/bitnami-docker-ghost#configuration
+##
+ghostProtocol: http
```
Comment (pchico83) [inline ghost/values.yaml]: set to https since Okteto Cloud always uses https

File: ghost/values.yaml [hunk 7]
```diff
@@ -0,0 +1,294 @@
...
+ghostEmail: user@example.com
+
+## Ghost Blog name
+## ref: https://github.com/bitnami/bitnami-docker-ghost#environment-variables
+##
+ghostBlogTitle: User's Blog
+
+## Set to `yes` to allow the container to be started with blank passwords
+## ref: https://github.com/bitnami/bitnami-docker-wordpress#environment-variables
+allowEmptyPassword: "yes"
```
Comment (pchico83) [inline ghost/values.yaml]: I would put this to "no" and remove it from `config.yaml`

File: ghost/values.yaml [hunk 8]
```diff
@@ -0,0 +1,294 @@
...
+  initialDelaySeconds: 30
+  periodSeconds: 5
+  timeoutSeconds: 3
+  failureThreshold: 6
+  successThreshold: 1
+
+##
+## External database configuration
+##
+externalDatabase:
```
Comment (pchico83) [inline ghost/values.yaml]: I think we can remove `externalDatabase`

File: ghost/values.yaml [hunk 9]
```diff
@@ -0,0 +1,294 @@
...
+      ##   GKE, AWS & OpenStack)
+      ##
+      # storageClass: "-"
+      accessMode: ReadWriteOnce
+      size: 8Gi
+
+## Kubernetes configuration
+## For minikube, set this to NodePort, elsewhere use LoadBalancer
+##
+service:
```
Comment (pchico83) [inline ghost/values.yaml]: This should be the Okteto Cloud definition, other configurations won't work in Okteto Cloud

File: ghost/values.yaml [hunk 10]
```diff
@@ -0,0 +1,294 @@
...
+resources:
+  requests:
+    memory: 512Mi
+    cpu: 300m
+
+## Configure the ingress resource that allows you to access the
+## Ghost installation. Set up the URL
+## ref: http://kubernetes.io/docs/user-guide/ingress/
+##
+ingress:
```
Comment (pchico83) [inline ghost/values.yaml]: Same, I would remove `ingress` all together

File: ghost/Chart.yaml [hunk 11]
```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+name: ghost
+version: 9.1.10
+appVersion: 3.8.0
+description: A simple, powerful publishing platform that allows you to share your stories with the world
+icon: https://pbs.twimg.com/profile_images/1183301397195833344/XRKixLnT_400x400.png
```
Comment (pchico83) [inline ghost/Chart.yaml]: @rlamana could you update the icon to follow our Okteto Cloud designs?

File: ghost/values.yaml (hunk 5 above)
Comment (genievot) [reply ghost/values.yaml]: ok great

**Comment by genievot** [comment]
thank you for adding the required changes @pchico83 .

File: ghost/values.yaml (hunk 8 above)
Comment (genievot) [reply ghost/values.yaml]: I just tested without externalDatabase, When the user uninstalls helm application then there is a database left that can be connected later on with the external database. But without it the chart gives an error that it is not able to connect MariaDB (Which inturn is then become external database)

Comment (pchico83) [reply ghost/values.yaml]: Makes sense, good point!

**Comment by pchico83** [comment]
@genievot would you mind splitting it in two different PRs? one PR for each app?

**Comment by genievot** [comment]
> @genievot would you mind splitting it in two different PRs? one PR for each app?

ok, that's nice.

**Comment by pchico83** [comment]
Closing this one in favor of a single PR for Drupal

**Pull request closed without merging**